packages/ses/src/lockdown.js

```diff
@@ -284,6 +285,7 @@ export const repairIntrinsics = (options = {}) => {
   addIntrinsics(tameMathObject(mathTaming));
   addIntrinsics(tameRegExpConstructor(regExpTaming));
   addIntrinsics(tameSymbolConstructor());
+  shimArrayBufferTransfer();
```

Comment:
Does this shim need to wait for `lockdown()`? Would it make sense to instead make it the behavior of importing `arraybuffer-transfer-shim.js` and do that in `index.js` instead?

(A current invariant among `-shim.js` modules exported at the top level is that they effect their behavior at initialization time, importing the behavior from a module without the `-shim` suffix.)

I ask because I could imagine ejecting this shim into a package, as I can imagine ejecting other layers.

Reply:
I looked, and moving it to `index.js` means that it could be bypassed:
- `package.json` exports `./lockdown-shim.js`
- ./lockdown-shim.js` just forwards to `./src/lockdown-shim.js`
- `./src/lockdown-shim.js` imports and calls `lockdown` without importing `../index.js`

So, for the current arrangement of things, no.

> I ask because I could imagine ejecting this shim into a package, as I can imagine ejecting other layers.

Once we unbundle `lockdown` itself into lots of vetted shims between `repairIntrinsics` and `hardenIntrinsics`, we should revisit this one as well. It will probably participate in that great unbundling.

Resolving.